serde-generate/tests/golang_generation.rs

```diff
@@ -125,3 +125,22 @@ fn test_golang_code_with_external_definitions() {
     let content = std::fs::read_to_string(source_path).unwrap();
     assert!(content.contains("foo.Tree"));
 }
+
+#[test]
+fn test_run_golang_runtime_tests() {
+    let mut runtime_mod_path = std::env::current_exe().unwrap();
+    runtime_mod_path = runtime_mod_path.to_owned();
+    runtime_mod_path.pop();
+    runtime_mod_path.pop();
+    runtime_mod_path.pop();
+    runtime_mod_path.pop();
+    runtime_mod_path.push("serde-generate/runtime/golang");
```

Comment:
I suspect `.join("../../../../etc")` works too

Reply:
not work. not sure what's going wrong :)